monkey/agent_plugins/exploiters/wmi/config-schema.json

```diff
@@ -0,0 +1,15 @@
+{
+    "required": [
+        "agent_binary_upload_timeout"
+    ],
+    "properties": {
+        "agent_binary_upload_timeout": {
```

Comment:
Is this the same timeout as `smb_download_timeout`?

Reply:
Yes